recipe/meta.yaml

```
@@ -102,10 +102,22 @@ outputs:
       run:
         - {{ pin_compatible("cuda-version", max_pin="x.x") }}
         - {{ pin_subpackage("libcublas", exact=True) }}
+        - cuda-cudart-dev_{{ target_platform }} {{ cuda_version }}.*
+        - cuda-crt-dev_{{ target_platform }} {{ cuda_version }}.*
```

```suggestion
        - cuda-cudart-dev_{{ target_platform }}
        - cuda-crt-dev_{{ target_platform }}
```
You should be able to drop the version pin here and let the `cuda-version` dependency align these.

---

Confirmed, and will fix this. Thanks.